openpower/patches/p8dtu-patches/hostboot-p8/0002-Support-130W-Turismo-Parts.patch

```diff
@@ -0,0 +1,40 @@
+From 3040a458385e82d7d9edf7b926fcdba641d0a048 Mon Sep 17 00:00:00 2001
+From: Jim Yuan <jim.yuan@supermicro.com>
+Date: Tue, 18 Oct 2016 16:12:21 -0700
+Subject: [PATCH 02/14] Support 130W Turismo Parts.
+
+Signed-off-by: Jim Yuan <jim.yuan@supermicro.com>
+---
+ src/usr/targeting/common/xmltohb/attribute_types.xml | 2 +-
+ src/usr/targeting/common/xmltohb/target_types.xml    | 2 +-
+ 2 files changed, 2 insertions(+), 2 deletions(-)
```

Comment:
Same comment here, should be able to put these into your p8dtu.xml instead

Reply:
If I move these settings to p8dtu.xml, how can I verify that the settings actually work? I hope I can find a way to see the changes in the build file to make sure.

Reply:
Dan any thought on Jim's response?

Reply:
You can always look at the generated system xml in output/host/usr/powerpc64le-buildroot-linux-gnu/sysroot/openpower_mrw_scratch/P8DSU_hb.mrw.xml to see the values of the attributes that were explicitly set.